frontend/src/data/runInfo.js

```diff
@@ -1,14 +1,5 @@
 import Summary from "./summary";
 
-const _DIALECT_URI_TO_SHORTNAME = {
-  "https://json-schema.org/draft/2020-12/schema": "Draft 2020-12",
```

Comment:
I'm guessing eslint flagged this as not being used right?

But I think that's probably actually incorrect in the code -- right now if you go to the homepage you see `Bowtie-draft202012` as the `<title>` but I think this mapping is/was meant to convert that into some more human readable name (even if it's only slightly different)